app/Http/Controllers/Table/FdbTablesController.php

```diff
@@ -182,14 +182,14 @@ public function formatItem($fdb_entry)
         if ($fdb_entry->created_at) {
             $item['first_seen'] = $fdb_entry->created_at->toDateTimeString();
         }
-
         if ($fdb_entry->port) {
-            $item['interface'] = Url::portLink($fdb_entry->port, $fdb_entry->port->getShortLabel());
-            $item['description'] = $fdb_entry->port->ifAlias;
-            if ($fdb_entry->port->ifInErrors > 0 || $fdb_entry->port->ifOutErrors > 0) {
-                $item['interface'] .= ' ' . Url::portLink($fdb_entry->port, '<i class="fa fa-flag fa-lg" style="color:red" aria-hidden="true"></i>');
+            $port = Port::where('port_id', $fdb_entry->port_id)->first();
```

Comment:
if somebody knows why I can't use the relation to get the port here as it used to be ... At least this fix removes the exception, but I feel bad doing this :)

Reply:
Something is fishy.  The relationship is being used above here.

Reply:
The left join with the device was pulling in a "port" field which then caused Eloquent to think the port relationship was already loaded.